### ISSUE TITLE
Make installer clear previous pool

### ISSUE THREAD

**Issue by pkgdemon** [trueos/trueos#51]
Currently we alternate between tank, and tank1 when installing from ISO.  For desktop this behavior may be desired for multiboot situations which is why it is the way it is now.  

However for the base OS (non desktop) which now uses text installer we do not offer multiboot we should clear, or destroy the previous pool to preserve a default "tank label".  FreeBSD's installer also behaves this way making a reinstall re-use the default "zroot" pool label unless the user specifies something else.  

All I am really saying here is we should always default to "tank" and blow the previous pool away since we are not worrying multi-boot scenarios in the foundation for building a desktop appliance on top of.  Instead alternating between tank, and tank1 behavior should remain available to be handled by the desktop installers but not the default for the core OS.  

In other words let the desktop installers alternate between tank, and tank1 but for our default we should just blow any previous pool away when a user selects a device for reinstall using the text installer.
